src/apps/explorer/ExplorerApp.tsx

```diff
@@ -32,21 +32,19 @@ const Home = React.lazy(
 )
 
 const HEADER = (
-  <Header
-    menu={
-      <Menu>
-        <li>
-          <Link to="/">Batches</Link>
-        </li>
-        <li>
-          <Link to="/trades">Trades</Link>
-        </li>
-        <li>
-          <Link to="/markets">Markets</Link>
-        </li>
-      </Menu>
-    }
-  />
+  <Header>
```

Comment:
ooohhhhh i see now. ignore previous comment on slotted things :sweat_smile:

Reply:
Already replied :D
Will make sure to ignore your reply if you ignore mine :)